integrations/jaxrs2/src/main/java/com/expedia/haystack/jaxrs2/feature/HaystackFeature.java

```diff
@@ -0,0 +1,30 @@
+package com.expedia.haystack.jaxrs2.feature;
+
+import javax.ws.rs.core.Feature;
+import javax.ws.rs.core.FeatureContext;
+import javax.ws.rs.ext.Provider;
+
+import com.expedia.haystack.jaxrs2.filters.ClientFilter;
+import com.expedia.haystack.jaxrs2.filters.ServerFilter;
+
+import io.opentracing.Tracer;
+
+@Provider
+public class HaystackFeature implements Feature {
+    private final Tracer tracer;
+
+    public HaystackFeature(Tracer tracer) {
+        this.tracer = tracer;
+    }
+
+    @Override
+    public boolean configure(FeatureContext context) {
+        if (context.getConfiguration().isEnabled(this.getClass())) {
```

Comment:
Shouldn't this be "if (!" (emphasis on the "not")?

Reply:
Yes!

Reply:
I understood that. From the URL you posted: "The responsibility of the feature is to properly update the supplied runtime configuration context and return true if the feature was successfully enabled or false otherwise." This logic says "if the context says that the configuration was already enabled then return false, otherwise register the filter (perhaps twice, once conditionally for server, but always for client, even if the configuration says server, which I find strange) and return true. Can you see how I find this confusing at the least and perhaps wrong?

Reply:
I changed that to be a conditional `switch` statement that should address the later half of your comment.